android/src/main/java/com/lwansbrough/RCTCamera/RCTCameraViewFinder.java

```diff
@@ -26,8 +29,9 @@
 import com.google.zxing.BarcodeFormat;
 import com.google.zxing.BinaryBitmap;
 import com.google.zxing.DecodeHintType;
+import com.google.zxing.LuminanceSource;
 import com.google.zxing.MultiFormatReader;
-import com.google.zxing.PlanarYUVLuminanceSource;
+import com.google.zxing.RGBLuminanceSource;
 import com.google.zxing.Result;
```

Comment:
In my pull request https://github.com/react-native-community/react-native-camera/pull/2766 for android I expose the [PlanarYUVLuminanceSource][1] to the `reactnative` developer with a method named `rectOfInterest`.

`rectOfInterest` allows the user to `set` the instance variables later then passed to [`PlanarYUVLuminanceSource`][1] constructor. 

```java
PlanarYUVLuminanceSource(byte[] yuvData, int dataWidth, int dataHeight, int left, int top, int width, int height, boolean reverseHorizontal) 
```

read more [here](https://github.com/react-native-community/react-native-camera/pull/2766#discussion_r398858333) (the linked comment is hidden, be sure to open it).

[1]: https://zxing.github.io/zxing/apidocs/com/google/zxing/PlanarYUVLuminanceSource.html